cmake/Extensions.cmake

```diff
@@ -115,7 +115,7 @@ macro(register_extension_linter target-name)
     get_property(extensions GLOBAL PROPERTY EXTENSION-LINTERS)
     set_property(GLOBAL APPEND PROPERTY EXTENSION-LINTERS "${target-name}")
     add_custom_target(${target-name}
-    COMMAND python ${CMAKE_SOURCE_DIR}/thirdparty/google-styleguide/run_linter.py -q -i ${CMAKE_CURRENT_LIST_DIR}/)
+    COMMAND python3 ${CMAKE_SOURCE_DIR}/thirdparty/google-styleguide/run_linter.py -q -i ${CMAKE_CURRENT_LIST_DIR}/)
```

Comment:
Is there a mainstream OS where `python` doesn't refer to python 3.x?

Reply:
Yes, for example Ubuntu 20.04.